tiny_dnn/core/kernels/conv2d_grad_op_avx.h

```diff
@@ -56,16 +92,25 @@ void avx_conv2d_5x5_back_kernel_one(const core::conv_params& params,
     const size_t in_padded_area = in_padded.area();
     float* pdelta_dst_org = &(*prev_delta)[0];
     const size_t  h_stride2 = params.h_stride * in_padded.width_;
-    static const __m256i imask = _mm256_setr_epi32(-1, -1, -1, -1, -1, 0, 0, 0);
-    static const __m256 mask = _mm256_castsi256_ps(_mm256_setr_epi32(-1, -1, -1, -1, -1, 0, 0, 0));
+    static const __m256i imask = _mm256_setr_epi32(
+        -1, -1, -1, -1, -1, 0, 0, 0);
+    static const __m256 mask = _mm256_castsi256_ps(
+        _mm256_setr_epi32(-1, -1, -1, -1, -1, 0, 0, 0));
     // propagate delta to previous layer
     if (w_stride == 1 && out.width_ >= 4) {
         const serial_size_t nblocks = out.width_ / 4;
-        for (serial_size_t inc = 0; inc < in.depth_; ++inc, pdelta_dst_org += in_padded_area) {
+        for (serial_size_t inc = 0;
```

Comment:
Maybe not yet, the type of `tiny_dnn::conv_params::Params::in' is `index3d<serial_size_t>` at the moment.